src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -30,6 +28,8 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWithNearbyCommentFilter"/>
```

Comment:
In the parent-pom, which I want to update to use the newer version of this build-resources, after the checkstyle rules are updated to work with said newer version. Updating this build-resources allows us to use these rules. Otherwise, we'll be blocked from updating that plugin because the rules won't work.